modules/installation-osp-creating-control-plane.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-creating-control-plane_{context}"]
+= Creating the control plane machines
+
+Create three control plane machines using Ignition files.
+
+.Prerequisites
+* The infrastructure ID from the installation program's metadata file is set as an environment variable (`$INFRA_ID`)
+* The `inventory.yaml` and `common.yaml` Ansible playbooks
+** If you need these files, copy them from *Creating network resources*
+* The 3 Ignition files created in *Creating control plane ignition files*
+
+.Procedure
+
+. Insert the following content into a local file called `04_control-plane.yaml`:
```

Comment:
just to reiterate, the user needs to copy the <INFRA_ID>-master-2-ignition.json files to the same location where 04_contro-plane.yaml file lives.